docker-compose.yaml

```diff
@@ -1,113 +1,8 @@
-version: "2"
+version: "3"
 services:
-
   docs-app:
     # to build this image locally, execute 'make docker-build'
     image: gsoci.azurecr.io/giantswarm/docs:latest
-
```

Comment:
since we are not using ES anymore for search in main docs, I will remove this to make it simpler for testing